tests_e2e/conftest.py

```diff
@@ -60,14 +61,14 @@ def _build_snowcli(venv_path: Path, test_root_path: Path) -> None:
 def _install_snowcli_with_external_plugin(
     venv_path: Path, test_root_path: Path
 ) -> None:
-    version = __about__.VERSION.replace("-", ".")
+    whl_file = _find_whl_file(os.listdir(test_root_path / f"../dist/"))
```

Comment:
I can approve it as temporary solution but I think we should consider hardcoding an expected file name here.

Reply:
I will search for some solution after release

Reply:
Ok, I forgot that we build snowcli in a venv, so I'm not sure we need any better solution :)